src/app/app.component.ts

```diff
@@ -22,16 +26,16 @@ import { DomUtilityService } from 'app/shared';
   template: `
     <ang-sky-banner *ngIf="showSkyBanner" [textValue]="skyBannerCopy" (closeBanner)="showSkyBanner = false"></ang-sky-banner>
     <div>
-      <div id="skip" tabindex="-1" aria-activedescendant="button">
+      <div id="skip" tabindex="-1">
         <button id="button" (click)="findMainContent()" (keyup.enter)="findMainContent()" tabindex="1" class="sr-only sr-only-focusable"> Skip to main content </button>
       </div>
       <nav-bar tabindex="-1"></nav-bar>
-    </div>
-    <main tabindex="-1">
```

Comment:
Will this break tabbing order / past accessibility work ? I 'm sure we can review that when looking at merging into stage branch